docs/knowledge/catalogs/test-catalog/text_generation/index.rst

```diff
@@ -4,10 +4,12 @@ Text generation tests
 - **Injections**
 
   - :func:`~.giskard.testing.tests.llm.test_llm_char_injection`
-  - :func:`~.giskard.testing.tests.llm.test_llm_prompt_injection`
+  - :func:`~.giskard.testing.tests.llm.test_llm_single_output_against_strings`
+  - :func:`~.giskard.testing.tests.llm.test_llm_output_against_strings`
 
 - **LLM-as-ajudge**
 
-  - :func:`~.giskard.testing.tests.llm.test_llm_output_against_requirement`
-  - :func:`~.giskard.testing.tests.llm.test_llm_single_output_against_requirement`
   - :func:`~.giskard.testing.tests.llm.test_llm_output_coherency`
+  - :func:`~.giskard.testing.tests.llm.test_llm_single_output_against_requirement`
+  - :func:`~.giskard.testing.tests.llm.test_llm_output_against_requirement`
+
```

Comment:
We're missing a new category (accuracy-related tests):

- https://github.com/Giskard-AI/giskard/blob/586e8537668777a8cedb1ce4a6749e5a8ba68e4b/giskard/testing/tests/llm/ground_truth.py#L16
- https://github.com/Giskard-AI/giskard/blob/586e8537668777a8cedb1ce4a6749e5a8ba68e4b/giskard/testing/tests/llm/ground_truth.py#L34